lib/Runtime/Language/AsmJsByteCodeGenerator.cpp

```diff
@@ -1944,32 +1944,27 @@ namespace Js
         }
 
         EmitExpressionInfo emitInfo;
-        // Arg3 - Value to Store
-        if (simdFunction->IsSimdStoreFunc())
+        if (simdFunction->IsSimdStoreFunc()) //Store
         {
+            // Arg3 - Value to Store. Builtin returns the value being stored.
             Assert(valueNode);
-            // Emit 3rd argument
-            valueInfo = Emit(valueNode);
-            if (valueInfo.type != simdFunction->GetArgType(2))
+            emitInfo = Emit(valueNode);
+
+            if (emitInfo.type != simdFunction->GetArgType(2))
```

Comment:
There is no issue/bug related to this. We uncovered it while enabling store dst opnd.